std/mmfile.d

```diff
@@ -124,7 +124,7 @@ class MmFile
 
         // Adjust size
         stat_t statbuf = void;
-        errnoEnforce(fstat(fd, &statbuf) == 0);
+        enforce(fstat(fd, &statbuf) == 0);
```

Comment:
All of these changes are bogus.

`errnoEnforce` is useful in that it catches `errno` and prints a useful error message depending on its value.